php_phongo_classes.h

```diff
@@ -242,18 +242,19 @@ extern zend_class_entry *php_phongo_connectiontimeoutexception_ce;
 extern zend_class_entry *php_phongo_writeexception_ce;
 extern zend_class_entry *php_phongo_bulkwriteexception_ce;
 
-extern zend_class_entry *php_phongo_type_ce;
-extern zend_class_entry *php_phongo_persistable_ce;
-extern zend_class_entry *php_phongo_unserializable_ce;
-extern zend_class_entry *php_phongo_serializable_ce;
 extern zend_class_entry *php_phongo_binary_ce;
 extern zend_class_entry *php_phongo_decimal128_ce;
 extern zend_class_entry *php_phongo_javascript_ce;
 extern zend_class_entry *php_phongo_maxkey_ce;
 extern zend_class_entry *php_phongo_minkey_ce;
 extern zend_class_entry *php_phongo_objectid_ce;
+extern zend_class_entry *php_phongo_persistable_ce;
```

Comment:
I don't think it makes sense to reorganise this. The specific classes' ce's should be together, but not mixed in with their interfaces. The rest of the CE's isn't alphabetised either.